### ISSUE TITLE
Missing Weapons

### ISSUE THREAD

**Issue by Fnetra** [ThomasJClark/elden-ring-weapon-calculator#30]
some weapons are missing. The ones I found so far are:
 zephyr blades
sword of the cosmos
molten sword
dragonkin seeker sword
guardian's spearblade

**Comment by ThomasJClark** [comment]
Those sound like Convergence weapons. Make sure you selected "The Convergence (mod)" in the top left dropdown

**Comment by Fnetra** [comment]
yep, already selected. the ones I listed were the only ones missing in the app that were still in the mod

**Comment by Fnetra** [comment]
btw, my mistake on zephyr blades, the missing ones are all straight swords for some reason.

**Comment by ThomasJClark** [comment]
Which weapons aren't showing up in that case? Can you post a screenshot including the weapon list and the filters/checkboxes?
